sbtc-cli/src/commands/mod.rs

```diff
@@ -0,0 +1,5 @@
+pub mod broadcast;
```

Comment:
Hate to do this...Mind adding forbid missing docs XD

Reply:
We should implement a Jacinta-bot to post these comments :joy: